app/Http/Controllers/Widgets/WorldMapController.php

```diff
@@ -44,7 +44,8 @@ public function __construct()
             'init_lng' => Config::get('leaflet.default_lng', 0),
             'init_zoom' => Config::get('leaflet.default_zoom', 2),
             'group_radius' => Config::get('leaflet.group_radius', 80),
-            'status' => '0,1',
+            'status' => '0,1', // Show all devices
+            'maintenance' => '0', // Hide devices under maintenance
```

Comment:
Any reason you are using textual numbers instead of just numbers?

Reply:
No, I guess not.